ext/ClimaOceanSpeedyWeatherExt/speedy_weather_exchanger.jl

```diff
@@ -13,6 +13,12 @@ import ClimaOcean.OceanSeaIceModels.InterfaceComputations:
     StateExchanger,
     interpolate_atmosphere_state!
 
+const OCRExt = Base.get_extension(Oceananigans, :OceananigansConservativeRegriddingExt)
+const SWGExt = Base.get_extension(SpeedyWeather, :SpeedyWeatherGeoMakieExt)
+
+get_cell_matrix(grid::SpeedyWeather.SpectralGrid) = SWGExt.get_faces(grid; add_nans=false)
+get_cell_matrix(grid::Oceananigans.AbstractGrid)  = OCRExt.compute_cell_matrix(grid, Center(), Center(), nothing)
```

Comment:
```suggestion
compute_cell_matrix(grid::Oceananigans.AbstractGrid)  = OCRExt.compute_cell_matrix(grid, Center(), Center(), nothing)
```

don't name mangle if you don't have to